styles/landing-page.css

```diff
@@ -0,0 +1,131 @@
+:root{
+    --primary-color: #DAA06D;
+}
+
+*{
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+body{
+    margin: 0;
+    padding: 0;
+}
+.main-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.nav-container{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: white;
+}
+.doc-container{
+    display: flex;
+}
+.title{
+    font-size: 2.5rem;
+    margin: 1.5rem;
+    margin-left: 2rem;
+}
+.menu-items ul{
+    margin: 0;
+    padding: 0;
+    display: flex;
+}
+.menu-items li{
+    list-style: none;
+}
+
+.menu-items li a {
+    text-decoration: none;
+    color: white;
+    font-size: 2rem;
+    margin: 1rem;
+    display: block;
+}
+.container{
+    position: relative;
+}
+
+.container .navigation
+{
+    position : relative;
+    width : 250px;
+    height: 500px;
+    background: white;
+    border-radius: 10px;
+    overflow:hidden;
+}
+
+.container .navigation ul{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
+.container .navigation ul li{
+    position: relative;
+    list-style: none;
+    width: 100%;
+}
+.container .navigation ul li a{
+    position: relative;
+    display: block;
+    width: 100%;
+    display: flex;
+    text-decoration: none;
+    color: var(--primary-color);
+    font-weight: 500;
+}
+.container .navigation ul li a .icon{
+    position: relative;
+    display: block;
+    min-width: 80px;
+    height: 60px;
+    line-height: 60px;
+    text-align: center;
+}
+.container .navigation ul li a .icon .fa{
+    font-size: 24px;
+}
+.container .navigation ul li a .title{
+    position: relative;
+    display: block;
+    height: 60px;
+    line-height: 60px;
+
+}
+.hero{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    height: 100vh;
+}
+.hero-title{
+    font-size: 10rem;
+}
+.hero-detail{
+    font-size: 2em;
+    font-weight: 100;
+    padding: 1rem;
+}
+.title-april{
+    color: var(--primary-color);
+}
+.title-ui{
+    background-color: var(--primary-color);
+    animation-name: change;
+    animation-duration: 2s;
+    animation-iteration-count: infinite;
+    color: white;
```

Comment:
variable naming can be used  for the color white because  it's repeating at many places  in your code